app/controllers/api/trainees_controller.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Api
+  class TraineesController < Api::BaseController
+    def show
+      trainee = Trainee.find_by(slug: params[:id])
+      if trainee.present?
+        render(json: trainee.to_json)
```

Comment:
I guess we'll need some kind of separate serialiser class here to control exactly what gets serialised. Analysis of what that should be is still work in progress. But do you think it's worth putting some kind of placeholder in here.

In the longer term it'll probably be version sensitive - different serialiser versions per API version.

Reply:
Yeah I think you're right, I'll put a placeholder in here.

Reply:
@stevehook I added a TraineeSerializer class and used that here instead. Internally it just calls `to_json` for now.